issue-12-The_number_of_planes_of_each_Airbus_Model

11michi11 opened the pull request.

11michi11 committed:
Prepared entity, repository and test for Count of Airbus planes by model

11michi11 committed:
Count of Airbus planes by model done

MatejMichalek commented on FlightsWebsite/src/js/planes.js:
```diff
@@ -0,0 +1,79 @@
+const url = 'https://flights-service-buki55n7ba-lz.a.run.app/planes/';
```
I am getting an error as 'url' constant is already declared in flights.js
![image](https://user-images.githubusercontent.com/31965407/82579865-b1456d80-9b8e-11ea-8109-01d942d39cfa.png)

we might store the URLs in some external file

11michi11 committed:
Overall JS improvement

Added common.js file for constants and common functions and core functions
Fixed issue with duplicate url variables
Fixed issue with multiple containers showing where they shouldn't
Fixed issue with disappearing containers
Fixed issue with reloading charts on selection of another drop down
Fixed issue with not resetting the drop down after changing selection

11michi11 committed:
Correct url
